doc/langdef.md

```diff
@@ -2100,7 +2123,8 @@ int("123") // 123 (if successful, otherwise an error)
 *   `string(uint) -> string` converts unsigned integer values to base 10
     representation
 *   `string(double) -> string` converts a double to a string
-*   `string(bytes) -> string` converts a byte sequence to a utf-8 string
+*   `string(bytes) -> string` converts a byte sequence to a UTF-8 string, errors
+    for invalid code points
```

Comment:
This is the behavior I've observed in `cel-go` and `cel-java` and it certainly seems correct to me.